op-challenger/challenger.go

```diff
@@ -48,8 +49,14 @@ func Main(logger log.Logger, cfg *config.Config) error {
 
 	for {
 		logger.Info("Performing action")
-		_ = agent.Act()
-		caller.LogGameInfo()
+		_ = agent.Act(context.Background())
+		status, _ := caller.GetGameStatus(context.Background())
+		if status != 0 {
+			caller.LogGameStatus()
+			return nil
```

Comment:
wait - should this return here?

Reply:
Yes. At this stage once the game is no longer in process the challenger should exit.

That won't last forever - once it supports multiple games it should carry on, but it doesn't support that yet.

Reply:
Yea so right now the challenger is run on a specific instance of a fault dispute game - with the address passed in as a cli flag. So, right now we want to exit the binary once that game is resolved. This will be changed/refactored in the future.